studentships.rst

```diff
@@ -33,10 +33,8 @@ Answers to FAQ
 - we particularly welcome `proposals <https://github.com/bio-tools/Studentships/blob/master/thematic_editing.pdf>`_ from `thematic editors <http://biotools.readthedocs.io/en/latest/editors_guide.html>`_.  
 
 ---------
-Proposals
+Former Proposals
```

Comment:
Same here, I'd just remove "former stuff/history" from there to make maintenance of this document more manageable